__tests__/TicketTest.js

```diff
@@ -0,0 +1,26 @@
+import { ERROR_MESSAGES } from "../src/constants/message";
+import { LOTTO } from "../src/constants/rules";
+import Ticket from "../src/Model/Ticket";
+
+describe("티켓 클래스 테스트", () => {
+  test("구입 금액이 숫자여야 한다.", () => {
+    expect(() => {
+      new Ticket("hi");
+    }).toThrow(ERROR_MESSAGES.ONLY_NUMBER);
+  });
+
+  test("구입 금액이 단위에 나눠 떨어져야한다.", () => {
+    expect(() => {
+      new Ticket("500");
+    }).toThrow(ERROR_MESSAGES.NOT_RIGHT_UNIT(LOTTO.PURCHASE_UNIT));
+  });
+
+  const AMOUNT_TEST_CASE = [
+    ["5000", 5],
+    ["132000", 132],
+  ];
+  test.each(AMOUNT_TEST_CASE)("구입 금액에 따른 티켓 개수 계산 test", (input, amount) => {
+    const ticket = new Ticket(input);
+    expect(ticket.getAmount()).toEqual(amount);
```

Comment:
```suggestion
    expect(ticket.getAmount()).toBe(amount);
```
원시 값을 비교하는 것 같은데 `toBe`가 아닌 `toEqual`을 사용하신 이유가 있으신가요??

Reply:
제가 Jest는 잘 몰라서.. 혹시 다른 점이 있나요..? 그냥 어디선가 봤어서 사용한 것 같아요 ㅎㅎ

Reply:
`toBe`: `object.is` 사용, 원시 값 비교에 적합
`toEuqal`: 모든 속성을 재귀적으로 비교(깊은 비교), 객체나 배열 비교에 적합
저는 이렇게 생각했던 것 같아요! 자세한건 공식문서 참고해보셔도 좋을 것 같아요🙃
https://jestjs.io/docs/expect#tobevalue
https://jestjs.io/docs/expect#toequalvalue

Reply:
와우 감사합니다! 참고해볼게요! 👍